src/topp/TextExporter.cpp

```diff
@@ -242,7 +242,14 @@ namespace OpenMS
   // stream output operator for a ProteinHit
   SVOutStream& operator<<(SVOutStream& out, const ProteinHit& hit)
   {
-    out << hit.getScore() << hit.getRank() << hit.getAccession()
+    // extract protein name from description
+    String protein_name = hit.getDescription();
+    if (protein_name.find("OS=") != std::string::npos)
```

Comment:
I don't like this unimod specific hack. There should be some other way to shorten the potentially long description string. I think it also adds an addtional space which is not very convinient.